src/System.IO.Compression.Brotli/System.IO.Compression.Brotli.csproj

```diff
@@ -30,6 +29,7 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(OS)' == 'OSX'">
     <SourceSystem>osx</SourceSystem>
+    <SourceArch></SourceArch>
```

Comment:
because, in osx we don't have a different architectures. So, we don't have additional folder layer(x64,x86) like in Linux and windows systems.